FormalConjectures/Paper/DegreeSequencesTriangleFree.lean

```diff
@@ -0,0 +1,168 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+Title: Degree sequences in triangle-free graphs
+Authors: P. Erdős, S. Fajtlowicz and W. Staton,
+Published in Discrete Mathematics 92 (1991) 85–88.
+-/
+
+open BigOperators
+open Classical
+
+/-- A sequence of natural numbers is **compact** if consecutive terms at distance
+`2` differ by `1`. -/
+def IsCompactSequence (d : ℕ → ℕ) : Prop :=
+  ∀ k, d (k + 2) = d k + 1
+
+namespace SimpleGraph
+
+variable {α : Type*} [Fintype α] [DecidableEq α]
+
+/-- The number of vertices of `G` having degree `d`. -/
+noncomputable def degreeFreq (G : SimpleGraph α) (d : ℕ) : ℕ :=
+  (Finset.univ.filter fun v : α => G.degree v = d).card
```

Comment:
Then here you can do
```suggestion
  #{v | G.degree v = d}
```

Reply:
Thank you,  I think that this change got merged as suggsted.